CMakeLists.txt

```diff
@@ -91,6 +94,35 @@ if(USE_CUDA)
   add_compile_definitions(__CUDA)
 endif()
 
+if(USE_ROCM)
+  set(CMAKE_CXX_STANDARD 11) 
+  SET(CMAKE_HIP_STANDARD 11)
+  # set(CMAKE_CXX_STANDARD_REQUIRED True) 
+  list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hip /opt/rocm)
```

Comment:
Does it offers a argument for customize ROCm dir?

Reply:
Cmake 3.21 mentioned the support for CXX=hipcc, but did not mention that there are new variable names, I think this part can be left as it is for the time being? 
[https://cmake.org/cmake/help/v3.21/release/3.21.html#new-features](url)
[https://rocmdocs.amd.com/en/latest/Installation_Guide/Using-CMake-with-AMD-ROCm.html](url)

Reply:
You're right. I suggest adding this url (https://rocmdocs.amd.com/en/latest/Installation_Guide/Using-CMake-with-AMD-ROCm.html) as comment for reference, do you think it's OK?